### ISSUE TITLE
 Failed to deploy ExApp 

### ISSUE THREAD

**Issue by ccgauvin94** [cloud-py-api/app_api#219]
### Describe the bug

I've installed AppAPI. I've also installed the docker proxy. I've successfully registered it as the daemon. If I open a terminal in the proxy container, I can successfully touch /var/run/docker.sock. When I go to the store and try to install any of the 5 currently available apps through the GUI, I get this error:

` Failed to deploy ExApp `

That's all I see in the GUI. In the NextCloud log, the only line that seems relevant is:

`[Sun Jan 28 03:31:25.883881 2024] [php:notice] [pid 51] [client 172.20.0.4:36082] cURL error 18: transfer closed with outstanding read data remaining (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://nextcloud_docker_socket_proxy:2375/v1.41/images/create?fromImage=ghcr.io/cloud-py-api/ai_image_generator_bot:2.0.0`

There's no log output from the proxy

### Steps/Code to Reproduce

Select "Deploy and Enable" on an ExApp. 

### Expected Results

Should install and enable. 

### Actual Results

Seemingly fails to download. 

### Setup configuration

Docker compose file for the socket proxy:
  nextcloud_docker_socket_proxy:
    image: ghcr.io/cloud-py-api/aa-docker-socket-proxy:release
    container_name: nextcloud_docker_socket_proxy 
    privileged: true
    restart: always
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    environment:
      - NC_HAPROXY_PASSWORD="some_secure_password"
    networks:
      - nextcloud_network

**Comment by bigcat88** [comment]
Thanks for reporting it, cause I totally missed about this(I saw such behaviour in one test production setup two days ago), but was ovehelmed with other stuff and bugs.

Will try to find out today why is the deploy process broken with Docker Socket Proxy

**Comment by bigcat88** [comment]
<img width="911" alt="image" src="https://github.com/cloud-py-api/app_api/assets/13381981/e4d1977f-3e76-488f-9685-ba2564a9ccf8">

<img width="1404" alt="image" src="https://github.com/cloud-py-api/app_api/assets/13381981/3518bd73-b9ce-4de5-a716-82d7dc6b2721">

<img width="252" alt="image" src="https://github.com/cloud-py-api/app_api/assets/13381981/0816e185-98ce-407d-a73b-016f2dd034d6">


@andrey18106 Any ides what is this? 
No errors in HaProxy, AppAPI backend successfully pulled image, created container and enabled ExApp, but in UI until I refresh page it is stuck. 
Docker-Socket-Proxy:1.0.1, AppAPI from last commit

**Comment by bigcat88** [comment]
@ccgauvin94 as a temporary workaround with Docker Proxy v1.0.1 and AppAPI 2.0.2 you can press "Deploy and Install" and after a few minutes close page. After some time(5-10-15min) App should be installed.
 
We'll rewrite this thing and make it normal a little bit later..

**Comment by ccgauvin94** [comment]
Thanks so much for looking into it!

I'm now getting a new error when I try that technique: 

```
colin@localhost:~/LocalAI> <134>Jan 29 03:02:09 haproxy[10]: 172.21.0.4:37414 [29/Jan/2024:03:00:39.248] docker_engine bk_docker_engine/dockersocket 0/0/0/2587/90539 200 150668 - - sD-- 1/1/0/0/0 0/0 "POST /v1.41/images/create?fromImage=ghcr.io/cloud-py-api/ai_image_generator_bot:2.0.0 HTTP/1.1"
-bash: syntax error near unexpected token `134'

```

Is that likely related?

**Comment by ccgauvin94** [comment]
Thanks so much for looking into this. I'll give that a try.

**Comment by bigcat88** [comment]
The should be fixed with AppAPI `2.1.0` - please check so we can close this

**Comment by ccgauvin94** [comment]
Working! Thanks for the fiox!
